diff --git a/include/xlsxwriter/common.h b/include/xlsxwriter/common.h
index 62f9c61..4856ab6 100644
--- a/include/xlsxwriter/common.h
+++ b/include/xlsxwriter/common.h
@@ -251,6 +253,8 @@ enum lxw_custom_property_types {
     if (error)                                  \
         return error;
 
+#if 0
+
 #define LXW_WARN(message)                       \
     fprintf(stderr, "[WARNING]: " message "\n")
 
@@ -300,6 +304,20 @@ enum lxw_custom_property_types {
        return;                                                             \
     }
 
+#else
+
+#define LXW_WARN(message) {}
+#define LXW_WARN_FORMAT(message) {}
+#define LXW_WARN_FORMAT1(message, var) {}
+#define LXW_WARN_FORMAT2(message, var1, var2) {}
+#define LXW_WARN_CAT_AXIS_ONLY(function) {}
+#define LXW_WARN_VALUE_AXIS_ONLY(function) {}
+#define LXW_WARN_DATE_AXIS_ONLY(function) {}
+#define LXW_WARN_CAT_AND_DATE_AXIS_ONLY(function) {}
+#define LXW_WARN_VALUE_AND_DATE_AXIS_ONLY(function) {}
+
+#endif
+
 #ifndef LXW_BIG_ENDIAN
 #define LXW_UINT32_NETWORK(n) ((((n) & 0xFF)       << 24) | \
                                (((n) & 0xFF00)     <<  8) | \
